config/leviton/DZPA1

```diff
@@ -0,0 +1,72 @@
+<!--
```

Comment:
Shouldn't this be a DZPA1.xml file? (missed the extension)

Reply:
Yes, changed to DZPA1.xml.